src/logic/addressBook/utils/v2-migration.ts

```diff
@@ -12,6 +12,8 @@ interface StorageConfig {
   namespaceSeparator: string
 }
 
+export const getSafesKey = (network: string) => `_immortal|v2_${network}__SAFES`
```

Comment:
This migration script will be deleted soon, please don't rely on it in the safe list.

Reply:
Shall I move this function to `SafeListItem` then?